src/main/java/com/conveyal/r5/analyst/WebMercatorGridPointSet.java

```diff
@@ -50,10 +54,15 @@ public WebMercatorGridPointSet(TransportNetwork transportNetwork) {
         LOG.info("Creating web mercator pointset for transport network with extents {}", transportNetwork.streetLayer.envelope);
 
         this.zoom = DEFAULT_ZOOM;
-        int west = lonToPixel(transportNetwork.streetLayer.envelope.getMinX());
-        int east = lonToPixel(transportNetwork.streetLayer.envelope.getMaxX());
-        int north = latToPixel(transportNetwork.streetLayer.envelope.getMaxY());
-        int south = latToPixel(transportNetwork.streetLayer.envelope.getMinY());
+
+        double latBuffer = SphericalDistanceLibrary.metersToDegreesLatitude(LinkedPointSet.MAX_OFFSTREET_WALK_METERS);
+        double lonBuffer = SphericalDistanceLibrary
+                .metersToDegreesLongitude(LinkedPointSet.MAX_OFFSTREET_WALK_METERS, transportNetwork.streetLayer.envelope.centre().y);
+
+        int west = lonToPixel(transportNetwork.streetLayer.envelope.getMinX() - lonBuffer);
```

Comment:
This could also be done with 

```
        Envelope envelope = new Envelope(transportNetwork.streetLayer.envelope);
        envelope.expandBy(lonBuffer, latBuffer);
```